### PR TITLE
Logout feature

### PR DESCRIPTION
**Pull request by donnelldebnam**
Closes Feature Request #5 

* Added logout button

### CODE REVIEW

File: src/main/webapp/index.jsp [hunk 1]
```diff
@@ -42,11 +43,19 @@
         Through our site, you will be able to connect with the world through online chat!
       </p>
       <ul>
-        <li><a href="/login">Login</a> to get started.</li>
-        <li>Go to the <a href="/conversations">conversations</a> page to
+        <% if(request.getSession().getAttribute("user") != null){ %>
```
Comment (dhgarrette) [inline src/main/webapp/index.jsp]: I'm not sure you meant to include this block in this PR.

File: src/main/webapp/WEB-INF/view/profile.jsp [hunk 2]
```diff
@@ -0,0 +1,51 @@
+<%--
```
Comment (dhgarrette) [inline src/main/webapp/WEB-INF/view/profile.jsp]: I'm not sure you meant to include this file in this PR.

File: src/main/webapp/WEB-INF/view/chat.jsp [hunk 3]
```diff
@@ -49,8 +49,9 @@ List<Message> messages = (List<Message>) request.getAttribute("messages");
   <nav>
     <a id="navTitle" href="/">CodeU Chat App</a>
     <a href="/conversations">Conversations</a>
-    <% if (request.getSession().getAttribute("user") != null) { %>
-      <a>Hello <%= request.getSession().getAttribute("user") %>!</a>
+      <% if (request.getSession().getAttribute("user") != null) { %>
```
Comment (dhgarrette) [inline src/main/webapp/WEB-INF/view/chat.jsp]: Some indentation stuff here.

File: src/main/webapp/WEB-INF/view/conversations.jsp [hunk 4]
```diff
@@ -29,7 +29,8 @@
     <a href="/conversations">Conversations</a>
     <% if (request.getSession().getAttribute("user") != null) { %>
       <a>Hello <%= request.getSession().getAttribute("user") %>!</a>
-    <% } else { %>
+      <a href="/logout.jsp">Logout</a>
+    <% } else{ %>
```
Comment (dhgarrette) [inline src/main/webapp/WEB-INF/view/conversations.jsp]: It looks like some of the upstream style fixes didn't make it into your code, e.g. the spaces after `else` in `} else {`.

**Comment by donnelldebnam** [comment]
All problems fixed... ready for new code review

**Comment by dhgarrette** [comment]
Thanks, Donell!  This is an awesome feature addition!!

**Pull request merged**